package.json

```diff
@@ -41,8 +41,8 @@
     "test:watch": "jest --watch"
   },
   "devDependencies": {
-    "@lavamoat/allow-scripts": "^2.3.1",
-    "@lavamoat/preinstall-always-fail": "^1.0.0",
+    "@lavamoat/allow-scripts": "^3.0.0",
```

Comment:
The PR title mentions `^3.0.1`. Seems like `^3.0.0` is the latest on NPM so maybe update the title to match that. Looks good to me otherwise.